source/frontend/src/features/contacts/contact/utils/contactUtils.ts

```diff
@@ -23,9 +23,14 @@ export const onValidateOrganization = (
   const errors = {} as any;
   try {
     // combine yup schema validation with custom rules
-    if (!hasEmail(values) && !hasPhoneNumber(values) && !hasAddress(values)) {
+    if (
+      !hasEmail(values) &&
+      !hasPhoneNumber(values) &&
+      !hasAddress(values) &&
+      values.isDisabled === false
```

Comment:
the `validatePerson()` method below has the following logic:
```tsx
if (
   ... &&
   (values.isDisabled === 'false' || values.isDisabled === false)
) {
...
}
```

But the `validateOrganization()` has a different logic: `values.isDisabled === false`
wouldn't it make sense to have the same check for disabled for person and organization?

Reply:
I agree, but I'll have to do a refactor on the logic we have now on contacts. I'll give it a go and update my PR.